en/docs/updates/best-practices.md

```diff
@@ -29,7 +29,14 @@ Here are the guidelines and recommendations to receive best update experience wi
     The best practices disseminating updates to production environment with no internet connection is as follows: <br>
     First [receive the updates](../update-tool/#update-commands-for-os) in a lower environment that is connected to the internet. Test the environment thoroughly.
     Perform security checks mandate by your organization and verify the updated pack. Last move the verified pack to the lockdown environment securely.
-    <br><br>
+    <br>
 5. Use a Configuration Management tool. <br>
    There are a myriad of benefits of using a Configuration Management Tool. WSO2 advise the use of Configuration Management Tool such as [Puppet and Ansible](../faq/#what-are-the-recommended-configuration-management-tools-to-deploy-configurations-to-client-nodes) to carry configuration changes flawlessly.
-   Read more on our [Continuous Update Document](../continuous-update).
\ No newline at end of file
+   Read more on our [Continuous Update Document](../continuous-update).
+   <br>
+6. Assess whether the environment getting the new updates has enough disk space before running the updates tool
+   It is important to have adequate disk space in the environment that gets updates. When the update the command is initiated the tool takes a full backup of the current product, load the applicable new updates and thereafter apply the new updates to the environment.
```

Comment:
When the update command is initiated, the tool takes a full backup of the current product, loads the applicable new updates and thereafter applies the new updates to the product pack